### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by Janbinkowski96**

### CODE REVIEW

File: .github/workflows/pythonapp.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: marker-selector
+name: Marker Selector deploy
```
Comment (szymon6927) [inline .github/workflows/pythonapp.yml:1]: Are you sure that it is a deploy process? I would like to name it `CD/CI` process

File: .github/workflows/pythonapp.yml [hunk 2]
```diff
@@ -24,7 +24,5 @@ jobs:
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-    - name: Test with pytest
-      run: |
-        # pip install pytest
-        # pytest
+    - name: Test with unittest
+      run: python testes.py
```
Comment (szymon6927) [inline .github/workflows/pythonapp.yml:28]: I think, there is a typo in the filename it should be `tests.py` but there is `testes.py`. Or am I missing something?

File: source/data_processing.py [hunk 3]
```diff
@@ -36,12 +37,12 @@ def load_data(self):
...
         if not condition_csv:
-            raw_data = pd.read_excel(self.data_in, index_col=0)
+            self.data_frame = pd.read_excel(self.data_in, index_col=0)
 
         if condition_csv:
-            raw_data = pd.read_csv(self.data_in, index_col=0)
+            self.data_frame = pd.read_csv(self.data_in, index_col=0)
 
-        return raw_data
+        return self.data_frame
```
Comment (szymon6927) [inline source/data_processing.py:45]: What is the reason of returning `self.` data? Any method in this class can access this. But if you really want then I would recommend to create a separate method for getting the data and separated one for setting. Eg.
```
def get_loaded_data(self):
    return self.data_frame


def load_data(self):
    don't return anything just settting the `self.data_frame`

File: .github/workflows/pythonapp.yml (hunk 1 above)
Comment (Janbinkowski96) [reply .github/workflows/pythonapp.yml:1]: ok

**Comment by szymon6927** [comment]
@Janbinkowski96 I would change ` tests_utils/` to `fixtures/` it's more readable and more convenient way

File: testes.py [hunk 4]
```diff
@@ -0,0 +1,283 @@
+import shutil
```
Comment (szymon6927) [inline testes.py:1]: The tests looks good to me. The only thing which in my opinion is worth to consider is to create a separate directory for `tests` and maybe divide this file into two smaller ones? So after all changes we will have something like this:
```
- tests/
    - data_processing_tests.py
    - model_tests.py
    - fixtures/
        - all fixtures needed to correctly run the tests
```

what do you thing?

Comment (Janbinkowski96) [reply testes.py]: sure it's good idea, it makes code more clearly

**Pull request merged**